### PR TITLE
update DOAP.java with default prefix

### PR DESCRIPTION
**Pull request by seralf**
added reccommended prefix for doap, as in [http://prefix.cc/doap](http://prefix.cc/doap)

This PR addresses GitHub issue: #866 .

Briefly describe the changes proposed in this PR:

* added PREFIX / NS constants for the vocabulary DOAP, to make it compliant with the others provided, which usually expose `NS(PREFIX,NAMESPACE)` constants

### CODE REVIEW

**Comment by jeenbroekstra** [comment]
Hi @seralf , thanks for this fix. Unfortunately, your contribution fails Eclipse IP validation. Please have a look at the [contributor guidelines](https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#legal-stuff), in particular the points on how to sign the Eclipse Contributors Agreement. Let us know if you need any help with any of it.

**Comment by jeenbroekstra** [comment]
Also: there's a compilation failure due to a missing import, can you fix that? It would be easiest if you _first_ fix the legal requirements: signing the ECA, and ensuring that you use the git signoff flag (`-s`), and then do the commit and push the fix for the missing import.

**Comment by seralf** [comment]
Hi Jeen sorry for that, I've missed that, I'll fix as soon as I'll have
time today.
Thanks very much for the instructions

2017-07-19 1:13 GMT+02:00 Jeen Broekstra <notifications@github.com>:

File: core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DOAP.java [hunk 1]
```diff
@@ -16,9 +16,22 @@
...
+	
+	/**
+	 * The DOAP namespace: http://usefulinc.com/ns/doap#
+	 */
 	public static final String NAMESPACE = "http://usefulinc.com/ns/doap#";
 
 	/**
+	 * An immutable {@link Namespace} constant that represents the DOAP namespace.
+	 */
+	public static final Namespace NS = new SimpleNamespace(PREFIX, NAMESPACE);
```
Comment (jeenbroekstra) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DOAP.java]: This doesn't compile: `SimpleNamespace` needs to be included in the imports of the `DOAP` class.

Comment (seralf) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DOAP.java]: sorry for that, I've included the import! thanks for the review

Comment (jeenbroekstra) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DOAP.java]: No worries. Do let us know when you've signed the ECA - the automatic script may not pick up on it, but we can manually verify it.

**Comment by seralf** [comment]
Hi Jeen, thanks
I've already signed the ECA!

2017-07-21 0:39 GMT+02:00 Jeen Broekstra <notifications@github.com>:

**Comment by jeenbroekstra** [comment]
Thanks @seralf ! I just confirmed that the ECA is signed. 

@jamesrdf I am not entirely sure that you have the necessary permissions to merge this PR (despite the failing check). Can you try?

**Comment by seralf** [comment]
Hi Jeen thanks for the support, even on this microscopic PR.
From the web interface I still see problems with a failing IP-validation,
but didn't see any button for merging

2017-07-24 1:20 GMT+02:00 Jeen Broekstra <notifications@github.com>:

**Comment by seralf** [comment]
thanks!

2017-07-24 17:15 GMT+02:00 James Leigh <notifications@github.com>:

**Pull request merged**